File: manager-web/src/main/resources/static/js/service/sellerService.js
//服务层
app.service('sellerService',function($http){


    //查找所有
    this.findAll = function () {
        return $http.get("./seller");
    }

    //分页查找
    this.findPage = function (pageNo, pageSize) {
        return $http.get("./seller/" + pageNo + "/" + pageSize);
    }

    //添加数据
    this.save = function (entity) {
        return $http.post("./seller", entity);
    }

    //修改数据
    this.update = function (entity) {
        return $http.put("./seller", entity);
    }

    //删除数据
    this.delete = function (selectIds) {
        return $http.delete("./seller?ids=" + selectIds);
    }

    //模糊查询数据
    this.search = function (pageNo, pageSize, entity) {
        return $http.post("./seller/search/" + pageNo + "/" + pageSize, entity);
    }
    //删除单行数据
    this.deleteOne = function (id) {
        return $http.delete("./seller/" + id);
    }

});

File: sellergoods-interface/src/main/java/com/gm/service/SellerService.java
package com.gm.service;

import com.gm.common.PageResult;
import com.gm.common.Result;
import com.gm.pojo.TbSeller;

import java.util.List;

public interface SellerService {
    Result findAll();

    PageResult findPage(Integer pageNo, Integer pageSize);

    Result findPage(TbSeller seller, Integer pageNo, Integer pageSize);

    Result saveSeller(TbSeller seller);

    Result updateSeller(TbSeller seller);

    Result deleteSeller(String[] ids);

    Result deleteOneSeller(String id);
}
